app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee.mdx

```diff
@@ -0,0 +1,277 @@
+
+If you're here to listen and chill? No problem! If you can't turn on your camera or aren't comfortable having your camera or even sound on? Feel free to turn them off! There is no obligation for you to have them on. We want you to be as comfortable as possible in our Coffees.
+
+We drop the schedule in the `#announcement` channel on Slack every Monday and the day of the Coffee. There will also be an announcement to join the Coffee in the `#vc-events` channel 10 minutes before the Coffee start, along with the link to join.
+
+### Joining Coffees
+
+- Head over to the `#vc-events` channel on Slack.
+- Click the "Join Event" button to join the Coffee.
+
+Now you will be directed to the Coffee's Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Co-Working with Other Members
+
+<LeadText>
+
+Do you want to work or learn with people around, need help and a pair program, or only want to hang out with other members? Thanks to **Dan Ott**, we have a `#co-working-room` channel linked to our Zoom, so our members can connect anytime! It's open 24/7!
+
+</LeadText>
+
+### Joining Co-Working Session
+
+- Head over to the `#co-working-room` channel on Slack.
+- Click the green button to start a new session or join the ongoing session.
+
+Now you will be directed to our Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Virtual Coffee Events
+
+<LeadText>
+
+Our members voluntarily hold events from members to members, and you're most welcome to join!
+
+</LeadText>
+
+### Events List
+
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/coffee-table-groups">
+  	Coffee Table Groups
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lunch-and-learns">
+  	Lunch & Learns
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lightning-talk">
+  	Lightning Talk
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/monthly-challenges">
+  	Monthly Challenges
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/hacktoberfest-initiative">
+  	Hacktoberfest Initiative
+  </Link>
+
+Every Monday, we drop the week's list of events in the `#announcement` channel on Slack. We also drop the schedule on the day of the events. There will be another announcement to join the events in the `#vc-events` channel 10 minutes before the events start, along with the link to join.
+
+You can find the complete schedule of the events on the <Link to="/events">Events</Link> page.
+
+### Joining Events
+
+- Head over to the `#vc-events` channel on Slack.
+- Click the "Join Event" button to join the event.
+
+Now you will be directed to the event's Zoom.
```

Comment:
Remove this line and put "- you will be directed to the event's Zoom room." in the list above